sale_quotation_number/models/sale_order.py

```diff
@@ -13,20 +13,22 @@ class SaleOrder(models.Model):
     @api.model_create_multi
     def create(self, vals_list):
         for vals in vals_list:
-            if self.is_using_quotation_number(vals):
-                company_id = vals.get("company_id", self.env.company.id)
-                sequence = (
-                    self.with_company(company_id)
-                    .env["ir.sequence"]
-                    .next_by_code("sale.quotation")
-                )
-                vals["name"] = sequence or "/"
+            # Check if it's a revision copy and maintain the original name
```

Comment:
@CRogos  Don't put such comment as this module is not aware of sale_order_revision one.

Reply:
Something more like `if the sale order name is not provided, ...`

Reply:
I thought as explanation why this is important to keep. But I agree that the unit test is the better choice.